keyboards/phantagom/oled_mpad/keymaps/default/keymap.c

```diff
@@ -0,0 +1,181 @@
+#include QMK_KEYBOARD_H
+
+enum Layer {
+    _NUMPAD = 0,        // Standard QWERTY layer
+    _MOUSE,          // Function key layer
+    _NAV,
+};
+#define _NP _NUMPAD
+#define _MS _MOUSE
+#define _NV _NAV
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  /*
+   * ,-------------------.
+   * | Fx | Fx | Fx | Fx |
+   * |----|----|----|----|
+   * | NL | /  | *  | -  |
+   * |----|----|----|----|
+   * | 7  | 8  | 9  |    |
+   * |----|----|----| +  |
+   * | 4  | 5  | 6  |    |
+   * |----|----|----|----|
+   * | 1  | 2  | 3  |    |
+   * |----|----|----| En |
+   * |    0    | .  |    |
+   * `-------------------'
+   */
+
+  [_NUMPAD] = LAYOUT_numpad_6x5(
+    MO(_MS), KC_MUTE, KC_VOLD, KC_VOLU,
+    KC_PSCR, KC_NUM, KC_PSLS, KC_PAST, KC_PMNS, 
+    TO(_NP), KC_P7,   KC_P8,   KC_P9,   
+    TO(_MS), KC_P4,   KC_P5,   KC_P6,   KC_PPLS, 
+    TO(_NV), KC_P1,   KC_P2,   KC_P3,   
+    KC_A, KC_P0, KC_PDOT,  KC_PENT  
+  ),
+
+  [_MOUSE] = LAYOUT_numpad_6x5(
+    _______, KC_MUTE, KC_VOLD, KC_VOLU,
+    KC_PSCR, KC_NUM, KC_BTN1, KC_BTN2, KC_BTN3, 
+    TO(_NP), KC_P7,   KC_MS_U,   KC_P9,   
+    TO(_MS), KC_MS_L,   KC_P5,   KC_MS_R,   KC_PPLS, 
+    TO(_NV), KC_P1,   KC_MS_D,   KC_P3,   
+    KC_A, KC_P0, KC_PDOT,  KC_PENT
+  ),
+
+  [_NAV] = LAYOUT_numpad_6x5(
+    MO(_MS), KC_MUTE, KC_VOLD, KC_VOLU,
+    KC_PSCR, KC_NUM, KC_PSLS, KC_PAST, KC_PMNS, 
+    TO(_NP), KC_HOME,   KC_UP,   KC_PGUP,   
+    TO(_MS), KC_LEFT,   _______,   KC_RIGHT,   KC_PPLS, 
+    TO(_NV), KC_END,   KC_DOWN,   KC_PGDOWN,   
+    KC_A, KC_INS, KC_DEL,  KC_PENT  
+  ),
+};
+
+void matrix_init_kb(void) {
+  // set CapsLock LED to output and low
+  setPinOutput(GP4);
+  writePinLow(GP4);
+  // set NumLock LED to output and low
+  setPinOutput(GP5);
+  writePinLow(GP5);
+  // set ScrollLock LED to output and low
+  setPinOutput(GP6);
+  writePinLow(GP6);
+}
+
+bool led_update_kb(led_t led_state) {
+    bool res = led_update_user(led_state);
+    if(res) {
+        writePin(GP4, led_state.num_lock);
+        writePin(GP5, led_state.caps_lock);
+        //writePin(B2, led_state.scroll_lock);
+    }
+    return res;
+}
+
+static  unsigned  int type_count = 0 ;
+ void  count_type ( void ) {
+    type_count++;
+}
+
+// Function to be called when the key is pressed/released 
+// If the function itself is not added, add each function, if it is, add only in `#ifdef OLED_ENABLE` 
+bool  process_record_user ( uint16_t keycode, keyrecord_t *record) {
+ #ifdef OLED_ENABLE 
+    if (record->event.pressed) {
+         count_type ();
+    }
+#endif 
+    return  true ;
+}
+
+void  oled_write_type_count ( void ) {
+     static  char type_count_str[ 7 ];
+     oled_write_P ( PSTR ( "Type count: " ), false );
+     itoa (type_count, type_count_str, 10 );
+     oled_write_ln (type_count_str, false );
```

Comment:
```suggestion
     oled_write_ln(get_u16_str(type_count, ` `), false );
```

Reply:
needed some more work, but this issue is addressed